### PR TITLE
[WEBDEV-486] Dropdown content switcher

### PR DESCRIPTION
**Pull request by vinternet-pds**

### CODE REVIEW

File: src/templates/prototypes/_toggles.pug [hunk 1]
```diff
@@ -44,3 +44,29 @@ block content
           dt Website:
           dd #[+link('#', 'www.conservatives.com')]
         p You may be able to discuss issues with your MP in person or online. Contact them for details.
+
+      h2 Radio Switch (content initally shown)
+      
+      +form('javascript:console.log("form submitted")', 'post')
```
Comment (jakemulley) [inline src/templates/prototypes/_toggles.pug]: random console.log?

File: src/javascripts/dropdown-switch.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            if (dropdownContentState === 'show') {
+              contentShow(dropdownContent);
+            } else if (dropdownContentState === 'hide') {
+              contentHide(dropdownContent);
+            }
+          }
+        };
+
+        // Loop radio group and grab clicked radio button
+        for (var x = 0; x < inputSwitchNamesArray.length; x++) {
```
Comment (jakemulley) [inline src/javascripts/dropdown-switch.js]: you use a forEach above and a for loop here - consistency?

File: src/templates/prototypes/_toggles.pug (hunk 1 above)
Comment (vinternet-pds) [reply src/templates/prototypes/_toggles.pug]: I followed on from the ways other dummy forms have been implemented - see form elements and search results pages. Happy to change to another approach if required.

File: src/javascripts/dropdown-switch.js (hunk 2 above)
Comment (vinternet-pds) [reply src/javascripts/dropdown-switch.js]: I know I originally did this differently for a reason but there is also no problem using foreach - amended and resubmitted.

**Pull request merged**